news/3907.bugfix

```diff
@@ -0,0 +1 @@
+Add a warning message if one of the tar file will be placed outside of target location causing potential security issue.
```

Comment:
FYI, as currently written this PR doesn't resolve issue #3907, so it shouldn't be marked as fixing the bug.

Reply:
Hi @cjerdonek i very much like to fail the process if issue is found but since this would be a breaking behavior, should we just try the error message first ?

Reply:
Nah, I'd missed that this was a security issue since I hadn't actually thought through the implications of the original behavior.

Directly switching to aborting such installs sounds like a better approach to me.

Reply:
The content of the news file needs to be updated.